src/pcre2_compile.c

```diff
@@ -10695,9 +11421,9 @@ if (!utf)
 is set we have to assume a numerical callout (4 elements) for each character
 plus one at the end. This is overkill, but memory is plentiful these days. For
 many smaller patterns the vector on the stack (which was set up above) can be
-used. */
+used. Include space for wrapping pattern in non-capturing parentheses. */
```

Comment:
A comment that explains better why the magic number 2 is needed, might be better IMHO

Reply:
This has been done, update not yet pushed.